fuse/cross_agency.py

```diff
@@ -147,6 +157,7 @@ def cross_match_officers_between_agencies(personnel, events, constraints):
                 ),
                 # but if two officers belong to the same attract constraint then give them the highest score regardless
                 AbsoluteScorer("attract_id", 1, ignore_key_error=True),
+                AbsoluteScorer("history_id", 1, ignore_key_error=True),
```

Comment:
Where is this `history_id` column generated?

Reply:
Typo. I will correct to `post_id`, which is generated in `match/post_ohr.py` as it is the field in which matched uids are concatenated. Do you think that `history_id` is a more appropriate name than `post_id`?

Reply:
I don't think the `post_id` is what I had in mind. In fact, I don't see why the uids need to be concatenated. The `history_id` could just be the frame index at the point where you have multiple uid columns. Then instead of concatenating uids, stack the frame instead. That way you will end up with 2 columns: `history_id` and `uid`. I hope that makes sense.

Also you need to make sure there are no uid that ends up in 2 different history records. It is quite easy to check.